### PR TITLE
Revert to commit 0ff9719

### PR DESCRIPTION
**Pull request by hugobho**
@kevgo 

This PR reverts all commits related to changing service role to service type for ExoCom's SERVICE_ROUTING

### CODE REVIEW

File: exorelay-javascript/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "exorelay",
-  "version": "0.20.2",
+  "version": "0.20.1",
```
Comment (kevgo) [inline exorelay-javascript/package.json]: leave the version at 2

File: exocom-server/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "exocom-dev",
-  "version": "0.16.2",
+  "version": "0.16.0",
```
Comment (kevgo) [inline exocom-server/package.json]: leave the version as is

File: exoservice-javascript/features/example-services/dependencies-contain-server-js/service.yml [hunk 3]
```diff
@@ -1,3 +1,3 @@
-type: dependency-contains-server
+name: dependency-contains-server
```
Comment (kevgo) [inline exoservice-javascript/features/example-services/dependencies-contain-server-js/service.yml]: Isn't `type` more correct here?

Comment (hugobho) [inline exoservice-javascript/features/example-services/dependencies-contain-server-js/service.yml]: Yes, these should all be `type`

**Pull request closed without merging**